components/form/signUpForm.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import { useFormik, Field } from 'formik'
+import { Button } from '../dataComponents/button'
+import { PageType } from '../Types'
+
+function validateForm(values: {
+  firstName: any
+  lastName: any
+  email: any
+  phone?: string
+  text?: string
+}) {
+  let errors: { [k: string]: any } = {}
+  if (!values.email) {
+    errors.email = 'Required'
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = 'Invalid email format'
+  }
+  if (!values.firstName) {
+    errors.firstName = 'Required'
+  }
+  if (!values.lastName) {
+    errors.lastName = 'Required'
+  }
+  if (!values.phone) {
+    errors.phone = 'Required'
+  } else if (
+    !values.phone.match(
+      /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im,
+    )
+  ) {
+    errors.phone = 'Invalid number'
+  }
+
+  if (errors.email || errors.firstName || errors.lastName || errors.phone) {
+    return errors
+  }
+  return null
+}
+
+export default function SignupForm({ data }: PageType) {
+  const dir = data.homepage.aboutTitle.includes('b') ? 'right-40' : 'left-40'
+  console.log(dir)
```

Comment:
Remove logs from the code when you're done. You can leave console.debug() if needed.

Reply:
console.log() removed